internal/cmd/validate.go

```diff
@@ -74,9 +84,14 @@ func (cmd *validateCmd) Run(args []string) int {
 }
 
 func (cmd *validateCmd) runInternal() error {
-	err := provider.Validate(cmd.ui)
+	err := provider.Validate(cmd.ui,
+		cmd.flagProviderDir,
+		cmd.flagProviderName,
+		cmd.flagProvidersSchema,
+		cmd.tfVersion,
+	)
 	if err != nil {
-		return fmt.Errorf("unable to validate website: %w", err)
+		return errors.Join(errors.New("validation errors found: "), err)
```

Comment:
Yes, it's to resolve a formatting issue. I wanted each validation error to be on a new line but with `fmt.Errorf()`, it was outputting:
```
        Error executing command: unable to validate website: matching resource for documentation file (resource2.md) not found, file is extraneous or incorrectly named
        missing documentation file for resource: scaffolding_example
```
instead of:
```
   Error executing command: validation errors found: 
   matching resource for documentation file (resource2.md) not found, file is extraneous or incorrectly named
   missing documentation file for resource: scaffolding_example
```